website/docs/d/edgegateway.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_edgegateway"
+sidebar_current: "docs-vcd-data-source-edgegateway"
+description: |-
+  Provides an edge gateway data source.
+---
+
+# edge gateway
+
+Provides a vCloud Director edge gateway directly connected to one or more external networks. This can be used to reference
```

Comment:
All new data source doc pages in this PR need the following update: we need to add "data source" somewhere in the sentence, because now it reads as if this is a resource.